.buildkite/build.rayci.yml

```diff
@@ -77,6 +77,7 @@ steps:
   - label: ":tapioca: build: doc"
     instance_type: medium
     commands:
+      - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
       - cd doc
       - FAST=True make html
```

Comment:
We should make this build the whole docs, by the way. IMO there's no sense in shortcutting on CI - it's our last chance we get to prevent broken code/docs from getting into `master`.

```python
make -C doc/ html
```

Would do the trick.